yml/archive/baselineSchema.yaml

```diff
@@ -1733,10 +1733,10 @@ tables:
       that the source is extended. extendedness = 0 implies a high degree of confidence
       that the source is point-like.
     mysql:datatype: FLOAT
-  - name: spuriousness
```

Comment:
Please revert all changes to this file: files in `archive/` are there for historical reference and should not be modified.

Reply:
Reverted the changes.